extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/Tools.stack/Analyse.pulldown/Get In Place Family Count.pushbutton/script.py

```diff
@@ -0,0 +1,42 @@
+"""
+Get a total count and types of In Place Family in the current document.
+Copyright (c) 2019 Jean-Marc Couffin
+https://github.com/jmcouffin
+--------------------------------------------------------
+PyRevit Notice:
+Copyright (c) 2014-2019 Ehsan Iran-Nejad
+pyRevit: repository at https://github.com/eirannejad/pyRevit
+"""
+
+__title__ = 'Get In Place Family count'
+__author__ = 'Jean-Marc Couffin'
+__contact__ = 'https://github.com/jmcouffin'
+__credits__ = 'http://eirannejad.github.io/pyRevit/credits/'
+__doc__ = 'Get a total count and types of In Place Family in the current document.'
+
+
+from pyrevit.framework import List
+from pyrevit import revit, DB
+
+famInstE = DB.FilteredElementCollector(revit.doc)\
```

Comment:
use [snake case](https://en.wikipedia.org/wiki/Snake_case) convention in python e.g. `family_instance_collector`